### PR TITLE
New /www/ files!

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by CmdrGree — APPROVED** [review]
seems fine

File: central_web_server/ui_files/survey.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    document.getElementById(id).setAttribute("class", newClass);
+}
+
+function loadSpecies(name, id, image, container) {    
+    var species = createHTMLElement({
+        tag: "div",
+        parent: container,
+        events: { "click": (() => species_onClick(id)) },
+        attributes: {
+            class: "species",
```
Comment (byran) [inline central_web_server/ui_files/survey.js]: class is a keyword in ECMAScript 2015 so I wouldn't use it without quotes

File: central_web_server/ui_files/index.js [hunk 2]
```diff
@@ -1,51 +1,102 @@
...
+    //TODO: only send required data.
+
+    window.location = "./survey.html";
 }
+
+function initialise() {
+    const speciesListsURL = "/api/getSpeciesLists";
+    getData(speciesListsURL, (result) => handleSpeciesListDownload(result));
+    document.getElementById("continue-button").addEventListener("click", continueButton_click);
+    //handleSpeciesListDownload(JSON.stringify(speciesLists));
```
Comment (byran) [inline central_web_server/ui_files/index.js]: Remove commented out code

File: central_web_server/ui_files/index.js [hunk 3]
```diff
@@ -1,51 +1,102 @@
...
+    } else if (invalidField == "touristIds") {
+        alert("Invalid Tourist IDs");
+        return;
+    }
+
+    window.localStorage.setItem("sessionId", sessionId);
+    window.localStorage.setItem("speciesList", speciesListId);
+    window.localStorage.setItem("touristIds", touristIds);
+    window.localStorage.setItem("species", JSON.stringify(speciesList));
+    //TODO: only send required data.
```
Comment (byran) [inline central_web_server/ui_files/index.js]: If this is true then you should do it

**Review by byran — CHANGES_REQUESTED** [review]

**Pull request merged**